plugins/woocommerce/package.json

```diff
@@ -103,7 +103,7 @@
 						"src/**/*.php",
 						"templates/**/*.php",
 						"tests/php/**/*.php",
-						"tests/unit-tests/**/*.php"
```

Comment:
I believe that we still want this to run when something in `tests/unit-tests` is changed, correct?

Also, should we update this for the other environments below (PHP 8.0, L-1, L-2, etc) as well?

Reply:
@jonathansadowski Nice catch, thank you. Restores those in https://github.com/woocommerce/woocommerce/pull/46247/commits/67ca464fa94dc844b2229233c5610b7ddc719fb4.